pages/incometax.vue

```diff
@@ -0,0 +1,130 @@
+        <i18n path="incometax.procedure.download.content" tag="p" class="paragraph-sm xl:paragraph">
+          <a
+          class="italic text-link"
+          :href="''"
+          target="_blank"
+          rel="noopener noreferrer"
+          >
+            {{ $t("incometax.procedure.download.form_label") }}
+          </a>
+        </i18n>
+      </paragraph>
+      <paragraph path="incometax.procedure.p1" :margin-bottom="false"/>
+      <ul>
+        <li class="mb-1 paragraph-sm xl:paragraph"> {{ $t("incometax.procedure.options.o1") }}</li>
+        <li class="mb-1 paragraph-sm xl:paragraph">
+          <i18n path="incometax.procedure.options.o2.content" tag="p" class="paragraph-sm xl:paragraph">
+            <a
+              class="italic text-link"
+              :href="'https://static.anaf.ro/static/10/Anaf/AsistentaContribuabili_r/telefoane_judete/Regiuni.htm'"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              {{ $t("incometax.procedure.options.o2.administrations_label") }}
+            </a>
+        </i18n>
+        </li>
+        <li class="mb-1 paragraph-sm xl:paragraph"> {{ $t("incometax.procedure.options.o3") }}</li>
+        <li class="mb-1 paragraph-sm xl:paragraph"> {{ $t("incometax.procedure.options.o4") }}</li>
+      </ul>
+      <paragraph :margin-bottom="false">
+        <i18n path="incometax.procedure.gdpr.content" tag="p" class="paragraph-sm xl:paragraph">
+          <a
+          class="italic text-link"
+          :href="''"
+          target="_blank"
+          rel="noopener noreferrer"
+          >
+            {{ $t("incometax.procedure.gdpr.document_label") }}
+          </a>
+        </i18n>
+      </paragraph>
+    </div>
+    <div>
+      <section-title path="incometax.deadline.title" />
+      <paragraph :margin-bottom="false">
+        <i18n path="incometax.deadline.p1.content" tag="p" class="paragraph-sm xl:paragraph">
+          <a
+          class="italic text-link"
+          :href="'http://cdep.ro/pls/proiecte/upl_pck.proiect?idp=18123'"
+          target="_blank"
+          rel="noopener noreferrer"
+          >
+            {{ $t("incometax.deadline.p1.link_label") }}
+          </a>
+        </i18n>
+      </paragraph>
+      <paragraph path="incometax.deadline.p2" :margin-bottom="false"/>
+    </div>
+    <div>
+      <section-title path="incometax.howto.title" />
+      <paragraph path="incometax.howto.p1" :margin-bottom="false"/>
+      <ul>
+        <li
+          v-for="(value, index) in $t('incometax.howto.list')"
+          :key="index"
+          class="mb-1 paragraph-sm xl:paragraph"
+        >
+          {{ value }}
+        </li>
+      </ul>
+      <paragraph path="incometax.howto.p2" :margin-bottom="false"/>  
+    </div>
+    <faq qanda="incometax.faq.q1" />
+    <faq qanda="incometax.faq.q2" />
+    <faq qanda="incometax.faq.q3" />
+    <faq qanda="incometax.faq.q4" />
+    <faq qanda="incometax.faq.q5" />
```

Comment:
Have a look on how you can use the `paragraphs` component here so that we don't introduce a new component and so that we don't have to repeat this line five times.
Using that the `faq` section would become an array that has elements with a common structure. You can find examples by searching how the component is used elsewhere.

Reply:
I don't think I can use `paragraphs` component as it is for this specific place, since I need also a section title.

Reply:
You are right, I was under the impression that you could use a `slot` instead of providing a path and just provide a template. It seems right now you can only provide a path.
So I suggest that we change `paragraphs` and `section-title` to accommodate something other than a path:

One simple implementation would be to add a new prop in 'paragraphs':
```typescript
  @Prop() literal: string | Array<string> | undefined
```
And then use it to overwrite the `path` in the `lines` computed property:
```typescript
const val = this.literal ? this.literal : this.$t(this.path)
```
And then we do the same for `section-title`:
```typescript
  @Prop() literal: string | undefined
```
In which case we would use a new computed property:
```typescript
  get title () {
    return this.literal ? this.literal : this.$t(this.path)
  }
```
And then `{{ title}}` in the template.  


All this together with using arrays in `incometax.faq` instead of *q1...qn* would allow us to then write:

```vue    
<div
  v-for="(qanda,i) in $t('incometax.faq')"
  :key="i"
>
  <section-title :literal="qanda.question" :margin-bottom="false" />
  <paragraphs :literal="qanda.answer" :margin-bottom="false" />
</div>
```

And we don't need new custom components or to repeat the *qn* lines. This also allows us to just add new questions in the *json* dictionaries and also to have a different number of questions per language.

This is a simple implementation but feel free to approach it differently, a better one would be to use an algebraic type for the type of 'path' which can distinguish between a path and a literal and so we would not need another prop.

Reply:
Please use this form
[Formular_230_2020.pdf](https://github.com/communitypoweredbydlot/community-website/files/5886633/Formular_230_2020.pdf)
Source: [ANAF](https://www.anaf.ro/anaf/internet/ANAF/asistenta_contribuabili/declararea_obligatiilor_fiscale/toate_formularele/!ut/p/a1/jZFRb4IwFIV_Cw-8ci8YkOytZk40xAyNEfpiCivIgoW0RZL9-gHbsi2ZzD713H735pxboBADFexaFkyXtWDVoKl3CuzACxzf2WAQIUbuYr1zV8-Ij24PJBMAer_7_YgsMQr3i-XaDnC1sz_78cYheF__BPCP_yPQSWRIMAITFjdAi6pOx3UlRKQzvwAqec4ll1Yr-_JZ60Y9mGhi13UWEyy3ZG3icDGxFJpLwbWJZEue-mrGFPtr1LlWGuIfE3r3CZ3fTD_vvSMcWTZ8JuyHIKrJT2TUW3bhkHzrsFSaCy6_KKVlq9XHGySGk2es1Uxy66WG5nI4xPjqVtcwr8r0jRDDeAe_fdpv/dl5/d5/L3dHQSEvUUtRZy9nQSEh/?fbclid=IwAR396j3uCasbg0z5e8hdsaUplTpw64dxsR1s1lW6mH9JFCLK5Fw1kgbasrQ#230n)

Reply:
@archblob Implemented your suggestions, works like a charm, removed the Faq component. Added the missing links.